modules/_labs/header/react/lib/Header.tsx

```diff
@@ -67,6 +67,7 @@ const HeaderShell = styled('div')<Pick<HeaderProps, 'variant' | 'themeColor'>>(
     WebkitFontSmoothing: 'antialiased',
     MozOsxFontSmoothing: 'grayscale',
     position: 'relative',
+    overflow: 'hidden',
```

Comment:
This prevents stuff from flowing out of the header when the viewport shrinks

Reply:
I don't think we can have this. It block the dropdowns, search autocomplete and avatar menu

Reply:
🤦‍♂ That's right, totally forgot about that.